web/samples_index/web/add_to_app.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Flutter samples</title>
+  <link href="styles.css" rel="stylesheet" media="screen">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script src="packages/mdc_web/material-components-web.min.js"></script>
+  <script defer src="description.dart.js"></script>
+</head>
+
+<body>
+  <div class="content">
+    <div class="navbar">
+      <a class="leading" href="/">
+        <img src="images/logos/logo_lockup_flutter_horizontal_wht.png" />
+        <span class="title">Samples</span> 
+      </a>
+      <div class="nav-items">
+        <a href="https://flutter.dev/">Flutter Home</a>
+        <a href="https://api.flutter.dev/">API Docs</a>
+      </div>
+    </div>
+    <div class="container">
+      <div class="description-title-row">
+        <h1>Add to App</h1>
+        <div class="type-label type-label-bordered">sample</div>
+      </div>
+      <p>By Flutter</p>
+      <div class="toolbar">
+        <div class="buttons">
+          <button class="mdc-button mdc-button--outlined" onclick="window.location.href = 'https://github.com/flutter/samples/tree/master/add_to_app';">
+      <div class="mdc-button__ripple"></div>
+      <i class="material-icons mdc-button__icon" aria-hidden="true">code</i>
+      <span class="mdc-button__label">Source Code</span>
+      </button>
+        </div>
+        <div class="tags-container">
+          <div class="tags-label">
+            <i class="material-icons">local_offer</i>
+            <span>Tags</span>
+          </div>
+          <div class="tags">
+            <a href="./#?search=tag%3Aadvanced">advanced</a>
+        <a href="./#?search=tag%3Asample">sample</a>
```

Comment:
Indenting looks a little weird in this file.

Reply:
there's some indentation code in `templates.dart` that might need some adjustments

Reply:
these generated files aren't committed to source control anymore, so I think this is a lower priority now.